docs/kyma-environment-broker/03-01-service-description.md

```diff
@@ -191,11 +192,19 @@ The mapping between the platform region and the provider region (Azure, AWS or G
  </details>
  </div>
 
-## Free plan
+## Own cluster plan
 
-Free plan allows you to install Kyma on Azure or AWS. The configurable provisioning parameters are the same as for the trial plan.
+These are the provisioning parameters for the `own_cluster` plan that you can configure:
```

Comment:
These are required parameters so I am not sure if the phrase `you can configure` is proper.